generate-specs.js

```diff
@@ -0,0 +1,21 @@
+const $RefParser = require('json-schema-ref-parser')
+const fs = require('fs')
+
+if (!fs.existsSync('./dist')){
+  fs.mkdirSync('./dist');
+}
+
+$RefParser
+  .dereference('./async_api/websocket_api.yaml')
+  .then((schema) => {
+    fs.writeFileSync('dist/async_api.json', JSON.stringify(schema, null, 2))
```

Comment:
are we not supporting API V2 by default? I believe all the validators have switched to rippled API v2 now

Reply:
This is just a script, we can add v2 — can you make a follow up PR?